<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register_UI</class>
 <widget class="QMainWindow" name="Register_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="registerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>注册|</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>昵称:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mailLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>邮箱:</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mailLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="sureToolButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
